QUICK_START.md

```diff
@@ -0,0 +1,153 @@
+with:
+```
+cd <impact-api-directory>
+git checkout development
+git pull
+```
+
+If you don't have impact-api checked out, but you do have the
+accelerate code base checked out, checkout the impact-api code
+next to the accelerate directory with:
+```
+cd <accelerate-api-directory>/..
+git clone https://github.com/masschallenge/impact-api.git
+cd impact-api
+```
+
+If you don't have the accelerate code base checked out, you will
+need it to get an initial database.  If you need this, then contact
+engineering@masschallenge.org.
+
+4. Run tests.  From the impact-api source directoy run:
+```
+make build
+make test
+```
+
+This will take a while, but all tests should pass.  Specifically the
+end of the output should look something like:
+
+```
+Ran 102 tests in 14.845s
+
+OK
+Destroying test database for alias 'default'...
+```
+
+4. Start server
+
+```
+make dev
+```
+
+Once the process completes (it will take a little while and the last
+line should sya "BUILD COMPLETE"), visit http://localhost:8000 in a
+browser to see the login screen.
+
+5. In a separate shell, load an accelerate database.  Assuming
+you have an up-to-date accelerate source code directory next to
+the impact-api directory, the following should work:
+
+```
+make dbload GZ_FILE=../accelerate/test_data/initial_schema.sql.gz
+```
+
+6. Create a new superuser account:
+
+```
+make superuser
+```
+
+This will ask you for an email address, full name, short name and a
+password.  For the purposes of this tutorial, we will assume the email
+address is "admin@example.com".
+
+7. Grant new account default access permissions
+
+```
+make grant-permissions PERMISSION_USER=admin@example.com PERMISSION_CLASSES=v0_clients,v1_clients
+```
+
+Rspond to the questions with "yes" or "y".
+
+8. Go to http://localhost:8000 and login as admin@example.com.  At
+this point you should be see a list of types of available accelerate
+objects.  Clicking on one of those links should give a you short list
+of some of the objects of the given type.  For example,
+http://localhost:8000/api/impact/Application/
+
+9. Next you need to create a client application.  Click the blue
+"Create An Application" button near the top of the page followed by
+"Register a new application".
+
+10. Give your new client application a name, for "Client type" select
+"Public" and for "Authorization grant type" select "Resource owner
+password-based".  "Redirect URIs" should not be given.  Click "Save".
+
+11. Copy the value under "Client id".  Note that it is a 40 character
+value that may not be completely visible.  However, double clicking it
+and then copying should get the entire value.
+
+12. Test functionally test the API, we recommend downloading Postman
+(https://www.getpostman.com/) and importing our testing collection.
+To import a collection into Postman go to File -> Import and select
+the "Import From Link" tab, insert the URL
+https://www.getpostman.com/collections/b9dcecd3190fcd10888d and click
```

Comment:
That collection is not organized to fit with documentation and it has bad calls in it like simpleuser. I would rather wait to add those calls to the initial list provided in the document.